build/common.project.props

```diff
@@ -86,15 +86,7 @@
     <DefineConstants>TRACE</DefineConstants>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(VisualStudioVersion)' == '14.0'">
-    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
-    <DefineConstants>$(DefineConstants);VS14</DefineConstants>
-    <VSSDKRoot>$(RepositoryRootDirectory)packages\Microsoft.VSSDK.BuildTools.14.3.25420</VSSDKRoot>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(VSSDKRoot)\tools</VSToolsPath>
-    <VsixOutputDirName>VS14</VsixOutputDirName>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(VisualStudioVersion)' == '15.0'">
+  <PropertyGroup>
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
```

Comment:
NuGet.VisualStudio.Client has bumped MinimumVisualStudioVersion to 16.0. Should this one be bumped too?  Should NuGet.VisualStudio.Client.csproj use this value?